test/ethereumRuntime.js

```diff
@@ -56,32 +69,23 @@ contract('Runtime', function () {
       const { code, pc, opcodeUnderTest } = getCode(fixture);
 
       it(fixture.description || opcodeUnderTest, async () => {
-        const initialStack = fixture.stack || [];
-        const initialMemory = fixture.memory || '0x';
+        const stack = fixture.stack || [];
+        const mem = fixture.memory || '0x';
         const { accounts, accountsCode } = encodeAccounts(fixture.accounts || []);
-        const callData = fixture.data || '0x';
-        const blockGasLimit = fixture.blockGasLimit || BLOCK_GAS_LIMIT;
-        const gasLimit = fixture.gasLimit || blockGasLimit;
+        const data = fixture.data || '0x';
+        const gasLimit = fixture.gasLimit || BLOCK_GAS_LIMIT;
         const logHash = fixture.logHash;
-        const callArgs = [
-          code,
-          callData,
-          [pc, 0, blockGasLimit, gasLimit],
-          initialStack,
-          initialMemory,
-          accounts,
-          accountsCode,
-          logHash,
-        ];
-        const res = await rt.initAndExecute(...callArgs);
+        const res = await rt.execute({ code, data, pc, gasLimit, stack, mem, accounts, accountsCode, logHash });
 
         if (fixture.result.stack) {
           assert.deepEqual(toStr(res.stack), fixture.result.stack);
         }
         if (fixture.result.memory) {
           assert.deepEqual(res.mem, fixture.result.memory);
         }
-        if (fixture.result.accounts) {
+        // TODO: re-enable once we support accounts again
```

Comment:
what are these accounts exactly? touched accounts? like contracts that have been called?

Reply:
Touched/changed accounts. I will re-enable them once I resolve the gas and stack depth issue :)